### ISSUE TITLE
[a11y]: ComboBox input element missing aria-controls

### ISSUE THREAD

**Issue by David-Yuen** [carbon-design-system/carbon#18776]
### Package

@carbon/react

### Browser

Chrome

### Operating System

MacOS

### Package version

1.76.0

### React version

18.3.1

### Automated testing tool and ruleset

IBM Equal Access Accessibility Checker

### Description

When the `ComboBox`'s dropdown list is expanded, the accessibility tool is throwing some errors. 

Based on the scan tool's description, my understanding is that the `input` field is missing `aria-controls` attribute. And the value for `aria-controls` should be the `id` value of the `ul` element that has `role="listbox"`..

Here is an example of the elements rendered by `ComboBox`:
<img width="540" alt="Image" src="https://github.com/user-attachments/assets/25832004-2eef-49ea-8f92-0519f4f967df" />

So i see the `ul` element that has `role="listbox"` has an `id` value of `downshift-:r5:-menu`. Then i go to the `input` field and add `aria-controls="downshift-:r5:-menu"`. If i re-scan after i add this, the scan error goes away
<img width="561" alt="Image" src="https://github.com/user-attachments/assets/6c7ed088-f37c-412a-b679-e27188172be5" />

### Reproduction/example

https://stackblitz.com/edit/github-2rrk6yyx?file=src%2FApp.jsx

### Steps to reproduce

Load the reproduction example page
Click the combobox to expand the dropdown menu
While the dropdown menu is expanded, run the scan
I see 2 errors, but main error is `"The 'aria-controls' attribute of the expanded combobox is missing"`
<img width="1779" alt="Image" src="https://github.com/user-attachments/assets/deb472b7-12ec-417f-9a31-16487826aef3" />

### Suggested Severity

None

### Code of Conduct

- [x] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon/blob/f555616971a03fd454c0f4daea184adf41fff05b/.github/CODE_OF_CONDUCT.md)
- [x] I checked the [current issues](https://github.com/carbon-design-system/carbon/issues) for duplicate problems

**Comment by AnvarPK** [comment]
Hi @tay1orjones , I'd love to take on this issue related to the combobox component in @carbon/react. Could you assign it to me if it's available? Thanks!

**Comment by David-Yuen** [comment]
The steps to reproduce are in the description, under the `Steps to reproduce` section

<img width="878" alt="Image" src="https://github.com/user-attachments/assets/757683b9-d47e-4d4b-8fd3-332418089602" />

**Comment by preetibansalui** [comment]
Hi @David-Yuen , I wasn't able to reproduce the issue in the provided Stackblitz or Storybook example. Can you please check again. Let me know if there are any specific steps or configurations needed to replicate it.

<img width="1697" alt="Image" src="https://github.com/user-attachments/assets/29f2e919-7000-4172-a78c-9190d993564f" />

I’ve also confirmed that the aria-controls attribute is present on the input element, as you mentioned.

<img width="1482" alt="Image" src="https://github.com/user-attachments/assets/0ead4174-ab41-458d-a54a-4681ee9aae6f" />

**Comment by kbpitula** [comment]
@preetibansalui any update? when this will be fix ?

**Comment by kbpitula** [comment]
@David-Yuen @preetibansalui  any update?

**Comment by heloiselui** [comment]
@David-Yuen, I believe this issue may have been resolved, as I'm no longer able to reproduce the error mentioned. Could you please confirm?

**Comment by ali-sadeghin** [comment]
We still have it, with "@carbon/react": "1.93.0"

**Comment by David-Yuen** [comment]
i havn't been able to update to latest carbon yet, waiting on some other fixes
